app/controllers/maps_controller.rb

```diff
@@ -12,9 +17,21 @@ def create
     if map.save
       map.center_image
       map.populate_characters
-      redirect_to campaign
+      campaign.update(current_map: map)
+      CampaignChannel.broadcast_current_map(campaign)
+      CampaignChannel.broadcast_map_selector(campaign)
+      head :ok
+    else
+      render partial: "form", locals: { map: map }, status: :bad_request
+    end
+  end
+
+  def destroy
+    map = current_user.maps.find(params[:id])
+    if map.destroy
+      redirect_to campaign, success: "Map successfully deleted."
     else
-      render :new, locals: { map: map }
+      redirect_to campaign, failure: "Map could not be deleted."
     end
   end
```

Comment:
Was this meant to be in this PR?

Reply:
No, thanks for catching it. I left this when I decided to break out editing and delete into a draft PR.

Removed in e7983c7.